.github/workflows/manual-sync.yml

```diff
@@ -0,0 +1,81 @@
+name: Sync from osu! upstream
+
+# Workflow dispatch can take inputs like any other github.event
+# As part of #5090 - we should evaluate adding inputs and outputs to refine this.
+on:
+  workflow_dispatch:
+    inputs:
+      force:
+        description: "Overwrite any changes in the target repository? (true/false)"
+        required: true
+        default: "false"
+
+jobs:
+  sync:
+    name: Execute Sync Task
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure Git to use your name
+        shell: bash
+        run: |
+          set -eo pipefail
+
+          #FIXME: We're using the noreply email of GitHub for now. Let's figure out a way to get the committer's email.
```

Comment:
I don't think this is meaningful. if you really want to go for it, try using `github.token` in conjunction with [`GET /users/{username}`](https://docs.github.com/en/rest/reference/users#get-a-user) and `json_pp`, but it's not worth the hassle.

Reply:
Oh the current fix works just as well so I'll remove this comment later